### PR TITLE
Add router, smooth scroll & increase font-weight

### PR DESCRIPTION
**Pull request by yoanad**

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -1,7 +1,46 @@
 .Header {
   height: 5em;
   display: flex;
-  justify-content: space-between;
+  /* justify-content: space-between; */
```
Comment (haensl) [inline src/components/Header/Header.css:4]: Once in source control, you can safely remove commented code.

File: src/components/Header/index.js [hunk 2]
```diff
@@ -1,17 +1,88 @@
...
-import "./Header.css";
+import React from 'react';
+import './Header.css';
+import { NavHashLink as NavLink } from 'react-router-hash-link';
 
-const Header = ({
+const Header = ({}) => {
+  const handleClick = (event) => {
+    event.target.classList.toggle('active');
+    console.log(event.target);
```
Comment (haensl) [inline src/components/Header/index.js:8]: Leftover `log`.

File: src/components/Header/index.js [hunk 3]
```diff
@@ -1,17 +1,88 @@
...
-      {/*fd fg dfgfd g*/}
-      {/*</div>*/}
+      <ul className="Header-Navigation" onClick={handleClick}>
+        <li>
+          <NavLink
+            to="/#about"
+            smooth={true}
+            isActive={() => {
+              return (
+                window.location.pathname + window.location.hash === '/#about'
```
Comment (haensl) [inline src/components/Header/index.js:20]: Imo you should turn this into a function, e.g.

```javascript
const isNavLinkActive = (path) => `${window.location.pathname}${window.location.hash}` === path;
```

and then use it on the `NavLink`s,

```javascript
<NavLink
  to="/#about"
  smooth={true}
  isActive={isNavLinkActive('/#about')}
/>
```

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -1,7 +1,46 @@
...
+
+.Header-Navigation {
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.Header-Navigation li {
+  list-style-type: none;
+  padding-right: 10px;
```
Comment (ZorianaPro) [inline src/components/Header/Header.css:20]: Can we use ems everywhere instead of px?

**Pull request merged**